src/desktop/modules/controllers/authController.ts

```diff
@@ -51,8 +51,21 @@ export default class AuthController {
             }
 
             const user = await permissionsHelper.getUser(userAuthDetails);
+            const managedUsers: any = await permissionsHelper.getManagedUsers(userAuthDetails);
```

Comment:
SDK should not have dependencies from outside of OpenRAP instead send all the users (main user + Managed user at once) to insert it